### ISSUE TITLE
[SF3] Biome Scanner shows no map

### ISSUE THREAD

**Issue by EnderofGames** [Eladkay/Scanner#20]
Minecraft version: 1.10.2
SF3 version: 3.0.10
Forge Version: 12.18.3.2281
Scanner version: 1.5.1 (also downgraded to 1.5, but resulted in crashes) 

I have attempted to look this problem up, but the only other person who shares it with me only posted their problem to the CurseForge page: 

>  As I can no longer replicate the crashes, I cannot provide a log. However, the GUI refuses to render during my testing.

>  My environment and Modpack are on 1.10.2.
On both, the Scanner GUI does not render. Instead, my character freezes, allowing my standard cursor to move about the screen until I click any portion of the screen or with any other input.

---(DitzyDubs, May 22, 2017)

The problem is relatively straight forward. I _**can**_ still remotely use the Terrain Scanner, I just don't actually know where I am clicking. Sound effects are played with the Biome Scanner, and the Biome Scanner uses power when these sounds are played. As you can also conclude, the Biome Scanner has had no problems receiving power, though it should be noted that I tried alternative generators and the like. 

Everything I have tried: 

- SSP and SMP. I am not the admin or host of the server, so my next test is getting someone to log on and have them try. I am hoping to have this by tomorrow night. 
- All four tiers of biome scanner, as well as the ultimate scanner through the terrain scanner exibit this problem. 
- I deleted and readded the mod. I also tried using the 1.5 version of the mod, found on the CurseForge 
site.
- Everytime I changed the mod, I made sure to remove all the scanners, then used creative to grab new ones. 

It possibly should be noted that I used the Curse/Twitch app for the SkyFactory 3 client.

Full gui before attempting to use scanner: (Everything is normal in this picture)
![2017-07-12_21 36 29](https://user-images.githubusercontent.com/17417666/28149438-d7702a48-674a-11e7-8a60-dcce2f0b4f83.png)
Gui after attempting to use scanner: (Note: screenshot function does not capture that the cursor shows up)
![2017-07-12_21 36 34](https://user-images.githubusercontent.com/17417666/28149441-da151a10-674a-11e7-935d-3f24060f2e9c.png)

**Comment by Eladkay** [comment]
Weird. Try using a more up to date version of FTBLib. Seems like it was already reported and it fixed itself.

**Comment by LatvianModder** [comment]
>1.10.2

lol

**Comment by EnderofGames** [comment]
I removed my current version of FTBLib, 3.6.3, and downloaded the newest release version, 3.6.5.

SSP and SMP still not working. No crashes, but still no GUI. Any other related mods that might affect it? 

Outlook at tonight for getting someone else on the server to try, still. I imagine it will work fine for them, but I feel like it is important info, still. 

> Seems like it was already reported and it fixed itself.

Are you so sure of that? I imagine the person on CurseForge just stopped attempting to use it, changed to a different version, or stopped playing altogether.

**Comment by EnderofGames** [comment]
Had another server user attempt to use the Biome Scanner. It crashed when he attempted to use the map. Reloaded and attempted again, same result.  

I remembered that I was not using _exactly_ SF3 as downloaded, I also have Optifine HD for 1.10.2 (U_D8). Upon deleting optifine and reloading SF3, attempting to use biome scanner crashes the game. Also repeated. When I redownloaded Optifine HD, the no-gui glitch reappears. 

I also grabbed a map loaded the area to see if that would help, and it did not. Here is the area, not updated since the first pictures I posted:

![2017-07-13_22 07 36](https://user-images.githubusercontent.com/17417666/28198978-6549903c-681f-11e7-8959-1dce1f524490.png)

I attempted to see if the problem would show up in other versions for me, so I used the twitch app to download _Scanner_. 1.10.2 and 1.11.2 both failed to startup. Both have Scanner, FTBlib, and CompactLayer, all auto-installed when I added _Scanner_ from the twitch app. 

Seeing this, I updated CompactLayer from online, finding the latest version and adding it to SF3. No change. 

Last I disabled the void-only worlds in SF3, and started a normal overworld to test there. The issue does not change. 

So perhaps Optifine is causing this issue, but without Optifine Minecraft is crashing on Biome Scanner use. Is there any other mods I can check to see if they are incompatible/need updating? Also, what do I need to test _Scanner_ out alone on Minecraft? 

Edit: I updated the FTButilities to 1.1x-3.6.5. Also no change. SF3 currently comes with 3.6.3.

**Comment by Eladkay** [comment]
Does updating Scanner and FTBLib to latest fix this?

**Comment by EnderofGames** [comment]
I have been having problems with SF3 that caused a delay, but today I started SkyFactory 3.0.13, updated _Scanner_ to v1.5.2, and it worked on SSP as soon as I tried. Our SMP server is down, but I have my fingers crossed for that. 

I did not update FTBLib in my first attempt, as the pack has the latest version already. 

I went to an older copy of SF3 v3.0.13, which was identical except it has _Scanner_ v1.5.1, and the old problem was present, it did not appear although I could interact. I am confident that _Scanner_ v1.5.2 has solved this issue. Both copies had Optifine 1.10.2 as a deviation from the standard SF3 pack, and it still worked with the scanner update.
